packages/origin-247-certificate/src/offchain-certificate/repositories/CertificateEvent/CertificateEventPostgres.repository.ts

```diff
@@ -128,11 +130,12 @@ export class CertificateEventPostgresRepository implements CertificateEventRepos
                 (
                     (attempt.attempts_count = 0) 
                     OR 
-                    (attempt.error IS NOT NULL AND attempt.attempts_count <= ${MAX_SYNCHRONIZATION_ATTEMPTS_FOR_EVENT})
-                )`
+                    (attempt.error IS NOT NULL AND attempt.attempts_count <= :attempts)
```

Comment:
This should be `<` i think, since it is compared with "max synchronizations".
So if attempts is `5` and max synchronization attempts is `5` it will still be queried.
And therefore "failed ids" should be `>=`